src/tools/wasm2wast.cc

```diff
@@ -72,8 +72,15 @@ static void parse_options(int argc, char** argv) {
         s_outfile = argument;
         ConvertBackslashToSlash(&s_outfile);
       });
+  // TODO(karlschimpf) Figure out why fold-exprs is on by default.
```

Comment:
Hmm, I don't know. The problem no longer happens. I guess I must of accidentally broke this, and then fixed the breakage after changing things.

Removing the no-fold-exprs flag.